src/dev-app/select/select-demo.html

```diff
@@ -107,17 +107,24 @@
     <mat-card-subtitle>Option groups</mat-card-subtitle>
 
     <mat-card-content>
-      <mat-form-field>
-        <mat-label>Pokemon</mat-label>
-        <mat-select [(ngModel)]="currentPokemonFromGroup">
-          <mat-optgroup *ngFor="let group of pokemonGroups" [label]="group.name"
-            [disabled]="group.disabled">
-            <mat-option *ngFor="let creature of group.pokemon" [value]="creature.value">
-              {{ creature.viewValue }}
-            </mat-option>
-          </mat-optgroup>
-        </mat-select>
-      </mat-form-field>
+      <p>
```

Comment:
Can these changes be reverted? The select demo is already pretty busy and the option centering input is something that is going away with the MDC-based select anyway.

Reply:
Sure, it was mostly for manual testing for myself and I left it there thought it could help others too.
I just removed it and force pushed the changes.